### PR TITLE
Feature/new pages

### PR DESCRIPTION
**Pull request by jeffvg**
#### Changes proposed in this Pull Request:
- add website draft document
- 
-

### CODE REVIEW

**Comment by jeffvg** [comment]
Kyle has more items, images and content to add

**Comment by huayunh** [comment]
Would be helpful if you plug the content in a spell checker too.

**Comment by jeffvg** [comment]
yes this should of been a draft PR. 
This content is all high level "daft items" taken off teams channel.

File: src/__configuration__/navigationMenu/navigation.tsx [hunk 1]
```diff
@@ -151,6 +151,16 @@ export const pageDefinitions: SimpleNavItem[] = [
...
+            {
+                title: 'Accessibility',
+                url: '/accessibility',
+                component: <MarkdownPage title={'Accessibility'} markdown={Docs.Design.Accessibility} wideLayout />,
+            },
+            {
+                title: 'For Websites',
+                url: '/websites',
+                component: <MarkdownPage title={'Designing for Websites'} markdown={Docs.Design.Websites} wideLayout />,
+            },
```
[the remark this answers is not in the document]
Comment (joebochill) [reply src/__configuration__/navigationMenu/navigation.tsx:163]: we'll get it ready

**Review by joebochill — APPROVED** [review]
I'm cool with the text content. Might want to get this into dev so we can get some other designer eyes on it while we're waiting for images.

**Pull request merged**